app/src/main/java/app/revanced/manager/ui/component/patches/OptionFields.kt

```diff
@@ -112,14 +120,21 @@ private interface OptionEditor<T : Any> {
     fun clickAction(scope: OptionEditorScope<T>) = scope.openDialog()
 
     @Composable
-    fun ListItemTrailingContent(scope: OptionEditorScope<T>) {
-        IconButton(onClick = { clickAction(scope) }) {
+    fun Dialog(scope: OptionEditorScope<T>)
+
+    @Composable
+    fun ListItemTrailingContent(
+        scope: OptionEditorScope<T>,
+        onClick: (clickAction: () -> Unit) -> Unit
```

Comment:
Handle clicks through the `OptionEditorScope` instead of adding a new parameter.